pulsar-transformations/src/main/java/com/datastax/oss/pulsar/functions/transforms/TransformFunction.java

```diff
@@ -319,6 +319,13 @@ private static TransformStep newComputeFieldFunction(ComputeConfig config) {
                 throw new IllegalArgumentException(
                     "Duplicate compute field name detected: " + field.getName());
               }
+              if (field.getType() == ComputeFieldType.DATE
+                  && ("value".equals(field.getName()) || "key".equals(field.getName()))) {
+                throw new IllegalArgumentException(
```

Comment:
Would the user get this feedback when they create the function via pulsar admin - or just later on in the logs when the function actually initializes?

Reply:
In the logs.